templates/cusdis/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Output,
+  randomPassword,
+  randomString,
+  Services,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const secret = randomString(32);
+  const databasePassword = randomPassword();
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `USERNAME=${input.username}`,
+        `PASSWORD=${input.password}`,
+        `NEXTAUTH_URL=https://${input.domain}`,
+        `DB_TYPE=pgsql`,
+        `DB_URL=postgres://postgres:${databasePassword}@${input.projectName}_${input.databaseServiceName}:5432/${input.projectName}?sslmode=disable`,
+        `JWT_SECRET=${secret}`,
+        `IS_HOSTED=true`,
+        `NODE_ENV=production`,
+        `HOST=https://${input.domain}`,
+        `SMTP_SENDER=Cusdis Notification<notification@${input.domain}>`,
```

Comment:
```suggestion
```

I don't think this is needed, unless you also add every other SMTP property to enable the notifications.

https://github.com/djyde/cusdis/blob/1fb82d5673dacee7961b87ee543d9a671c645e8e/public/doc/features/notification.md?plain=1#L25